pantry-finder-api/.env.beta

```diff
@@ -0,0 +1,4 @@
+DB_NAME=ssm:db-name
```

Comment:
This syntax allows us to pull sensitive values from aws parameter store at build time.